src/main/java/service/FruitValidator.java

```diff
@@ -0,0 +1,19 @@
+package service;
+
+import java.util.Arrays;
+
+public class FruitValidator implements Validator {
+    private static final int OPERATOR_INDEX = 0;
+    private static final int NAME_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+    private static int EXPECTED_ARRAY_LENGTH = 3;
+
+    @Override
+    public boolean validate(String[] lineParse) {
+        return lineParse.length == EXPECTED_ARRAY_LENGTH
+                && Arrays.asList(Operations.values())
+                .contains(Operations.valueOf(lineParse[OPERATOR_INDEX]))
+                && !lineParse[NAME_INDEX].equals("")
```

Comment:
`Operations.valueOf()` will throw exception if there is no such enum
so the logic with enums is overcomplicated

also, let's better use `string.length() instead of `equals()`